source/includes/_energy-ratings.md

```diff
@@ -30,10 +31,95 @@ curl https://api.gresb.com/api/v1/energy_ratings
 
 ### Create, Update and Remove Energy Rating
 
-To add an energy rating to an asset, send a request to `/entities/{entity_id}/assets/{asset_id}` using PATCH method. Energy ratings are posted annually as outlined in the data dictionary. Each energy rating has to have an ID and a size specified in order to be valid. Energy ratings can be updated with the corresponding `energy_rating_id` and removed by setting the related fields to `null`.
+Previously, participants could submit only one energy rating per asset. As per the 2024 Standard Changes, asset data entry form will be accomodating the submission of **multiple energy ratings per asset**, facilitating more accurate reporting especially meant for residential units.
+
+Unlike the previous structure where energy ratings were reported under the objects "energy_rating_id" and "energy_rating_size" in the annual_data array, they will now be nencapsulated within their own array, outside the `annual_data`, similar to Building Certifications.
+
+To add an energy rating to an asset, send a request to `/entities/{entity_id}/assets/{asset_id}` using PATCH method. Energy ratings are no longer posted annually as outlined in the data dictionary. Each energy rating has to have an ID and a size specified in order to be valid.
+
+New format of the payload is exemplified with the sample JSON on the side:
+
+Energy ratings can be updated with the corresponding `energy_rating_id` and removed by setting the related fields to `null`.
+
+```shell
+curl https://api.gresb.com/api/v1/entities/{{entity_id}}/assets/{{asset_id}}
+```
+> PATCH Request:
 
-Beware that an asset cannot have more than one energy rating per year.
+```json
+{
+    "energy_ratings": [
+        {
+        "energy_rating_id": 57,
+        "size": 123
+        },
+        {
+        "energy_rating_id": 66,
+        "size": 456
+        }
+    ]
+}
+```
 
-Following is how part of your request body should look like:
-`{ "annual_data": [{ "year": 2022, "energy_rating_id": 57, "energy_rating_size": 50 }] }`
+> Response:
 
+```json
+[
+    {
+        "gresb_asset_id": {asset_id},
+        "country": "NL",
+        "state_province": "North Holland",
+        "city": "Amsterdam",
+        "address": "Barbara Strozzilaan 374",
+        "lat": 52.335915637320596,
+        "lng": 4.889755053768094,
+        "partners_id": 123456,
+        "construction_year": 1800,
+        "asset_ownership": 55,
+        "certifications":    [
+            {
+            "id": {unique_id},
+            "certification_id": 598,
+            "name": "BCA Green Mark/Existing Buildings",
+            "level": "Platinum",
+            "size": 230,
+            "_validations": {"errors": {}}
+            }
+        ],
+        "energy_ratings": [
+            {
+            "energy_rating_id": 57,
```

Comment:
depending on the `id` discussion, we do well also to add a response example

because the response will include `id` to be able to delete it afterwards